src/main/resources/migrations/0006_add_fields_to_webhook_messages.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add-field-to-webhook-messages
+
+ALTER TABLE webhook_messages
+ADD COLUMN resource_external_id VARCHAR(26),
+ADD COLUMN resource_type VARCHAR;
+
+--rollback DROP column resource_external_id, DROP column resource_type;
```

Comment:
This should be in a separate PR (we deploy database changes separately from code changes).

Reply:
@alexbishop1 OK, I have changed it now to update the existing migration, as we are currently re-creating the DB during each deploy.